cfme/tests/cloud_infra_common/test_cloud_init_provisioning.py

```diff
@@ -13,6 +13,7 @@
 from cfme.infrastructure.pxe import get_template_from_config
 from cfme.markers.env_markers.provider import providers
 from cfme.utils import ssh
+from cfme.utils.bz import BZ
```

Comment:
This import is what's causing travis to fail, `BZ` is in `cfme.utils.blockers` - along with GH. The bugzilla wrapper classes are what's in `cfme.utils.bz`